components/logins/src/db.rs

```diff
@@ -645,6 +645,50 @@ impl LoginDb {
         Ok(exists)
     }
 
+    pub fn verify_logins(&self, encdec: &dyn EncryptorDecryptor) -> Result<()> {
```

Comment:
I hate to nitpick the names too much, but "verify_foo" doesn't imply stuff might be deleted. Maybe `delete_undecryptable_records_for_remote_replacement()` just to keep the theme going :)

I honestly don't care too much though, whatever :)